assets/src/stories-editor/blocks/amp-story-cta/edit.css

```diff
@@ -125,6 +125,7 @@ div[data-type="amp/amp-story-cta"] .amp-overlay {
 	position: absolute;
 	z-index: 10;
 	margin-top: -0.5em;
+	top: 100%;
```

Comment:
This seems like a rather substantial change. How can that be inconsequential for the LTR view?

Reply:
I'd have to ask the CSS gods 😄